package.json

```diff
@@ -8,8 +8,7 @@
     "test": "jest",
     "test:cov": "jest --coverage",
     "test:watch": "jest --watchAll",
-    "cypress": "cypress run",
-    "cy:open":"cypress open",
+    "cypress": "cypress open",
```

Comment:
Check up with @IsaiahRn on this. Why did you delete `cypress run`. It should be left for it to run on CLI.

Reply:
Solved.